addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudAPI.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.voice.voicerss.internal.cloudapi;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Locale;
+import java.util.Set;
+
+/**
+ * Interface which represents the functionality needed from the VoiceRSS TTS
+ * service.
+ * 
+ * @author Jochen Hiller - Initial contribution
+ */
+public interface VoiceRSSCloudAPI {
+
+	/**
+	 * Get all supported locales by the TTS service.
+	 * 
+	 * @return A set of @{link {@link Locale} supported
+	 */
+	Set<Locale> getAvailableLocales();
+
+	/**
+	 * Get all supported audio formats by the TTS service.
```

Comment:
Pretty unspecific as it returns a String - no one will have a clue, what this String is supposed to look like without having a closer look at the code.